frontend/packages/console-shared/src/components/badges/Badge.scss

```diff
@@ -9,6 +9,9 @@
   &.pf-c-label {
     background-color: #d93f00;
     border-radius: var(--pf-global--BorderRadius--sm);
+    :before {
+      border-color: transparent;
```

Comment:
Hey @jcaianirh, can you comment on why this is needed?

Reply:
@spadgett sure, The pf label css changed slightly to work better on a gray background: https://github.com/patternfly/patternfly/pull/3817. But we customize the css for the label in the tech preview badge css so it conflicts with the pf css change. The tech preview badge is squared off, and the pf has a capsule shape, so you can see a capsule border without this change (@dtaylor113 caught this while testing pf-2021.2 with openshift):
![image (2)](https://user-images.githubusercontent.com/35978579/108133643-e978ce80-7082-11eb-8459-1fc86d8589d6.png)

Reply:
Thanks. I don't know the history here, but I wonder if we shouldn't be using the PF label style as-is.

cc @christianvogt @andrewballantyne @serenamarie125

Reply:
Not sure why we used a `Label` component originally 🤔 ... predates me.

I'll log a tech debt item to swing back and try to clean this up. It's odd usage of the `Label` component then to just override its styles.

Thanks for the heads up!